src/Http/Response/Format/Json.php

```diff
@@ -91,12 +98,29 @@ protected function morphToArray($value)
     /**
      * Encode the content to its JSON representation.
      *
-     * @param string $content
+     * @param array $content
      *
      * @return string
      */
-    protected function encode($content)
+    protected function encode(array $content)
```

Comment:
The reason I added `array` type-hinting for `$content`, is that I noticed this argument is currently marked as `string` in docbock. This is not correct, because `encode` method can be called only form number of methods in `Json` and `Jsonp` classes, and they all _explicitly_ pass array into it, and _never_ pass string. All current tests are green. Is there any possible situations, uncovered by tests, when non-array (object?...) argument can be passed into `encode` method? If the answer is "no" it not a breaking-change. (I guess it is no). Another way is to get rid of type-hinting in signature and change docblock to `@param mixed $content` or so, but I think it not a good decision, because "explicit is better than implicit". :)